xml/vk.xml

```diff
@@ -17450,6 +17450,13 @@ typedef void <name>CAMetalLayer</name>;
                 <enum value="&quot;VK_NV_extension_433&quot;"           name="VK_NV_EXTENSION_433_EXTENSION_NAME"/>
             </require>
         </extension>
+        <extension name="VK_MESA_rect_list" number="434" type="device" author="MESA" contact="Clément Guérin @libcg" supported="vulkan">
+            <require>
+                <enum value="1"                                         name="VK_MESA_RECT_LIST_SPEC_VERSION"/>
+                <enum value="&quot;VK_MESA_rect_list&quot;"             name="VK_MESA_RECT_LIST_EXTENSION_NAME"/>
+                <enum offset="0" extends="VkPrimitiveTopology"          name="VK_PRIMITIVE_TOPOLOGY_RECT_LIST_MESA"/>
+            </require>
+        </extension>
```

Comment:
Since the extension modifies logical device behaviour, it also needs a feature structure as per the [Feature Structures](https://github.com/KhronosGroup/Vulkan-Docs/blob/3a94dab07f8359af9130d3a344107cb422691397/style/extensions.txt#L1181) section of the style guide